### ISSUE TITLE
Data Request

### ISSUE THREAD

**Issue by cuncunsama** [yuanzhengthu/handDrawnPAAImages#3]
Would you be willing to share the PPA dataset you've created using the DOVE method?

**Comment by yuanzhengthu** [comment]
> Would you be willing to share the PPA dataset you've created using the DOVE method?

Sure, I’d be happy to share it for the progress of our field! How would you like me to provide the dataset to you?

**Comment by 464hee** [comment]
Google Drive？

**Comment by yuanzhengthu** [comment]
Yeah, after I upload the data, I'll send you a link. I can share everything about this work—just don't forget to cite me! Lol

I believe the next generation of AI will be trained on the images generated like this. Feel free to hit me up at yuanzheng@berkeley.edu so I don't miss any messages. Cheers!

**Comment by yuanzhengthu** [comment]
Hey 464hee,

Sorry for the delayed response—I've been swamped with work lol. But I’m excited to discuss this method further, especially if you’re into AIGC for medicine.

I’ve uploaded my dataset on Zenodo. You can check it out here: [Zenodo Dataset Link](https://zenodo.org/records/12802601).

Looking forward to some nerdy discussions!

Best regards,
